File: 24-tests/example_go/averegator/averegator.go
package averegator

import (
	"project/summator"
)

// Функция для подсчета среднего значения элементов в массиве
func Average(numbers []int) float64 {
	sum := summator.Sum(numbers)
	return float64(sum) / float64(len(numbers))
}

File: 2024_238/22-tests/example_c/averegator.c
// averegator.c
#include "averegator.h"

double average(int *numbers, int size) {
    if (size == 0) {
        return 0;
    }
    
    int sum = 0;
    for (int i = 0; i < size; i++) {
        sum += numbers[i];
    }
    
    return (double) sum / size;
}

